app/uk/gov/hmrc/agentservicesaccount/views/main.scala.html

```diff
@@ -93,7 +94,7 @@
         additionalHeadBlock = Some(head(hasTimeout)),
         additionalScriptsBlock = Some(scripts),
         beforeContentBlock = Some(beforeContent),
-        mainContentLayout = if(fullWidth) Some(fullWidthMainContent(_)) else None
+        mainContentLayout = if(fullWidth) Some(twoThirdsLayout(_)) else None
```

Comment:
this should be `if(fullWidth) Some(fullWidthMainContent(_)) else Some(twoThirdsLayout(_))`